test/sqllogictest/chbench.slt

```diff
@@ -259,7 +259,7 @@ ORDER BY ol_number
 ----
 %0 =
 | Get materialize.public.orderline (u19)
-| Filter (datetots(#6) > 2007-01-02 00:00:00)
+| Filter (2007-01-02 00:00:00 < datetots(#6))
```

Comment:
Stupid question: Don't we want the constants to come second? There is some visual appeal for it, but, most importantly, will that have any impact on downstream or future transformations/optimizations?

Reply:
The order is `column references`, `literals`, `unary operations`, `binary operations`, etc. This won't have any impact. Currently there is no transformation relying on expression ordering, since this normalization didn't exist.